zipline/pipeline/loaders/blaze/core.py

```diff
@@ -1220,6 +1220,36 @@ def bind_expression_to_resources(expr, resources):
     })
 
 
+def load_raw_data(assets, dates, data_query_time, data_query_tz, expr,
```

Comment:
This feels more like a utility than something that should be in core since it is not used by the core loader.

Also, could you add a docstring to document the use of this function?